modules/virt-importing-vm-datavolume.adoc

```diff
@@ -77,8 +77,8 @@ spec:
         storageClassName: local
       source:
         http: <1>
-          url: "https://download.fedoraproject.org/pub/fedora/linux/releases/33/Cloud/x86_64/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2" <2>
-          secretRef: "" <3>
+          url: <https://mirror.arizona.edu/fedora/linux/releases/35/Cloud/x86_64/images/Fedora-Cloud-Base-35-1.2.x86_64.qcow2> <2>
+          secretRef: <endpoint-secret> <3>
```

Comment:
Possibly out of scope: I am confused about what we're trying to communicate here. Should this be `<endpoint_secret>`, as in a generic value to be replaced? I think it's especially confusing because this page also includes references to "the `endpoint-secret.yaml` file".

(also applies to the next file)

Reply:
It's the "name" of the Secret. Can I use this example if I take out the `<>`?

Reply:
Yes!